gems/smithy-client/lib/smithy-client/plugins/logging.rb

```diff
@@ -18,6 +18,7 @@ class Logging < Plugin
           :log_level,
           default: :info,
           doc_type: Symbol,
+          doc_default: ':info',
```

Comment:
This isn't working as expected - I think this should be `"':info'"` otherwise I think we may just have a small bug with option default generation. In v3 this handles correctly.